building/glass/aptbranch.py

```diff
@@ -73,3 +43,34 @@ def export_key(keyid, armor=False):
     if not result.strip():
         raise Exception("empty result from gpg for keyid: '%s'" % keyid)
     return result
+
+
+class AptBranchConfig:
+    def __init__(self, branch: str):
+        if not os.path.exists(CONFIG_PATH):
+            raise Exception("cannot find apt branch config under %s" % CONFIG_PATH)
+        branches_config = validate.load_validated(CONFIG_PATH, CONFIG_SCHEMA_NAME)
+        self.config = None
+        for branch_config in branches_config["branches"]:
+            if branch_config["name"] == branch:
+                self.config = branch_config
+                break
+
+        if self.config is None:
+            raise Exception("no config found for %s in %s" % (branch, CONFIG_PATH))
+
+        check_signing_key(self.signing_key)
+
+    @property
+    def name(self) -> str:
+        return self.config["name"]
+
+    @property
+    def signing_key(self) -> str:
+        return self.config["signing-key"]
+
+    @property
+    def apt_url(self):
+        if "apt-url-prefix" in self.config:
+            return self.config["apt-url-prefix"] + self.name
+        return self.name
```

Comment:
To make this functionality more obvious, I would either include an `else` clause or use the pattern of 'assign to local variable; mutate variable if condition holds; return local variable'